stories/09.Overlay/NightingaleOverlay.stories.ts

```diff
@@ -0,0 +1,46 @@
+import { Meta, Story } from "@storybook/web-components";
+import { html } from "lit-html";
+import "../../packages/nightingale-navigation/src/index.ts";
+import "../../packages/nightingale-saver/src/index.ts";
+
+export default {
+  title: "Components/Utils/Overlay",
+  argTypes: {
+    for: {
+      options: ["all", "navigation", "sequence", "div"],
+      control: { type: "radio" },
+    },
+  },
+} as Meta;
+
+const Template: Story<{
+  for: string;
+  label: string;
+}> = (args) => {
+  return html`
```

Comment:
Is it possible to show that scrolling is happening in the story? Since the label `Use [CTRL] + scroll to zoom` isn't being displayed it means the overlay is not active, but thought it would be nice to show the wheel/scroll does something to the thing under the overlay when it's supposed to.

Reply:
yeah good point... I'll wrap the story on a div with a fix size that requires scrolling!